packages/aws-amplify-react-native/src/InAppMessaging/components/hooks/index.ts

```diff
@@ -1,3 +1,4 @@
 export { default as useMessage } from './useMessage';
 export { default as useMessageImage, ImageDimensions } from './useMessageImage';
 export { default as useMessageProps } from './useMessageProps';
+export { default as useOrientation, DeviceOrientation } from './useDeviceOrientation';
```

Comment:
Probably want to export as the same name
```suggestion
export { default as useDeviceOrientation, DeviceOrientation } from './useDeviceOrientation';
```

Reply:
Thanks for catching that